src/components/_General/ExplorerLink.tsx

```diff
@@ -31,19 +35,23 @@ const TxButtons = styled.div`
 
 const ExplorerLink = ({
   txid,
+  txidColor,
   type,
   postfix = '',
 }: {
   txid: string;
+  txidColor?: string;
   type?: string;
   postfix?: string;
 }) => {
   return (
     <ExplorerLinkRoot>
-      <TxId>{txid}</TxId>
+      <TxId txidColor={txidColor}>{txid}</TxId>
       <TxButtons>
         <CopyToClipboard textToCopy={txid} color={Colors.WHITE} />
-        <OpenInExplorer link={`${links.explorers[TICKER]}/${type || 'tx'}/${txid}/${postfix}`} />
+        <OpenInExplorer
+          link={links.explorers[TICKER](`${type || 'tx'}/${txid}${postfix && `/${postfix}`}`)}
```

Comment:
I'd love if this was extracted into a `buildExplorerLink` method and attached to a variable so we keep this logic separate